package.json

```
@@ -68,7 +68,7 @@
     "test:tgz-check": "./scripts/tgz-check.sh",
     "test:attribution-check": "./scripts/attributions-check.sh",
     "test:merge-coverage": "nyc merge ./tests/coverage ./tests/merged-coverage/coverage.json",
```

This is what I meant to suggest:

```suggestion
    "test:merge-coverage": "nyc report -t ./tests/coverage --report-dir ./tests/merged-coverage/ --reporter json --reporter text --reporter lcov",
```

This would generate the coverage report in three different formats using the intermediary coverage data from the shard runs. The check-coverage command does not require any changes